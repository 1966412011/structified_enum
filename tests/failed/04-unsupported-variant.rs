use structified_enum::structify;

#[structify]
enum Foo {
    A()
}

#[structify]
enum Bar {
    Bar {}
}

fn main() {}
