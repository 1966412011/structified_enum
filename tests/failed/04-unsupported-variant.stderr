error: unsupported variant
 --> tests/failed/04-unsupported-variant.rs:5:5
  |
5 |     A()
  |     ^

error: unsupported variant
  --> tests/failed/04-unsupported-variant.rs:10:5
   |
10 |     Bar {}
   |     ^^^
